frontend/src/app/uploads/DeviceLookup/DeviceSearchResults.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import { DeviceType } from "../../../generated/graphql";
+import Button from "../../commonComponents/Button/Button";
+
+interface SearchResultsProps {
+  devices: DeviceType[];
+  // TODO: fix this type
+  setSelectedDevice: any;
+  shouldShowSuggestions: boolean;
+  loading: boolean;
+  dropDownRef?: React.RefObject<HTMLDivElement>;
+  queryString?: string;
+}
+
+const DeviceSearchResults = (props: SearchResultsProps) => {
+  const {
+    devices,
+    setSelectedDevice,
+    shouldShowSuggestions,
+    loading,
+    dropDownRef,
+    queryString,
+  } = props;
+
+  let resultsContent;
+
+  if (loading) {
+    resultsContent = <p>Searching...</p>;
+  } else if (devices.length === 0) {
+    resultsContent = (
+      <div
+        className={
+          "display-flex flex-column flex-align-center margin-x-7 margin-y-2"
+        }
+      >
+        <div className="margin-bottom-105">
+          No device found matching <strong>{queryString}</strong>
```

Comment:
copy nit: period after this? we have it in our other 'no match' copy when searching for patients in the queue page or the results page